lib/style/pipes.ex

```diff
@@ -28,7 +28,7 @@ defmodule Styler.Style.Pipes do
   # we're in a multi-pipe, so only need to fix pipe_start
   def run({{:|>, _, [{:|>, _, _} | _]}, _} = zipper, ctx), do: {:cont, zipper |> check_start() |> Zipper.next(), ctx}
   # this is a single pipe, since valid pipelines are consumed by the previous head
-  def run({{:|>, meta, [lhs, {fun, _, args}]}, _} = zipper, ctx) do
+  def run({{:|>, _, [lhs, {fun, meta, args}]}, _} = zipper, ctx) do
```

Comment:
the pipe's meta contained a `newline: 1`. this chooses to keep the meta from the function call instead
an alternative fix would be to keep the pipe meta but delete the newline, but this seemed like less bruhaha to me

Reply:
I think this makes logical sense as well, since you're just sucking up the pipe input into the function arguments, as opposed to turning the pipe into a function call.